opencga-app/src/main/java/org/opencb/opencga/app/migrations/v3/v3_0_0/OrganizationMigration.java

```diff
@@ -434,16 +445,83 @@ protected void run() throws Exception {
             // Set organization counter, owner and authOrigins
             orgCol.updateOne(Filters.eq("id", organizationId), Updates.combine(
                     Updates.set("_idCounter", counter),
-                    Updates.set("owner", organizationId),
+                    Updates.set("owner", ownerId),
                     Updates.set("configuration.authenticationOrigins", authOrigins)
             ));
         }
 
+        // If the user didn't want to use the userId as the new organization id, we then need to change all the fqn's
+        if (!this.organizationId.equals(this.userId)) {
+            changeFqns();
+        }
+
         // Skip current migration for both organizations
         catalogManager.getMigrationManager().skipPendingMigrations(ParamConstants.ADMIN_ORGANIZATION, opencgaToken);
         catalogManager.getMigrationManager().skipPendingMigrations(organizationId, opencgaToken);
     }
 
+    private void changeFqns() throws CatalogDBException {
+        this.dbAdaptorFactory = this.mongoDBAdaptorFactory;
+        String date = TimeUtils.getTime();
+
+        // Change project fqn's
+        for (String projectCol : Arrays.asList(OrganizationMongoDBAdaptorFactory.PROJECT_COLLECTION,
```

Comment:
For projects, should update the "project.internal.datastores.variant" field if empty.
The expected value could be obtained from, but that method might not work as it uses catalog.
```
VariantStorageManager.defaultDataStore(catalogManager, project, token)
```